README.md

```diff
@@ -78,6 +78,10 @@ if you want to help.
 You can find our current feature and bug priorities for forthcoming
 releases [in the projects section](https://github.com/ethereum/solidity/projects).
 
+## Guides
+
+Refer the link for further examples: https://solidity-by-example.org/
```

Comment:
This probably belongs to the documentation. Perhaps in the resources section? https://docs.soliditylang.org/en/v0.8.6/resources.html

@franzihei What do you think?

Reply:
Ok let me know which section i should include the example part

Reply:
https://github.com/ethereum/solidity/blob/develop/docs/resources.rst probably under the section generic?

Reply:
ok i will create a new pull request with this changes made thank you @hrkrshnn for guidance